cmd/start.go

```diff
@@ -43,13 +46,22 @@ func Start(repoName string, tp node.Type) *cobra.Command {
 				return err
 			}
 
-			return start(cmd, tp, repo)
+			u, err := url.Parse(cmd.Flag(cfgAddress).Value.String())
+			opts := make([]node.Options, 0)
+			if err == nil && u.Scheme != "" && u.Host != "" {
+				opts = append(opts, node.WithRemoteClient(u.Scheme, u.Host))
+			}
+
```

Comment:
👍🏻 for proper `url.Parse` usage. 

Pls extract parsing into a separate small helper function. Also, we should error to the user if he passes a malformed URL and ideally check if port presents inside parsed url.Host.